src/main/java/org/docstr/gradle/plugins/gwt/GwtBasePlugin.java

```diff
@@ -89,20 +89,19 @@ public void apply(final Project project) {
         .setDescription("Classpath for GWT SDK libraries (gwt-dev, gwt-user)");
     allGwtConfigurations = project.files(gwtConfiguration, gwtSdkConfiguration);

```

Comment:
This is not used internally anymore, but since there is a getter, I didn't want to make breaking changes by removing this.

Reply:
Yes, seems not being used. But ok to keep it for now.